src/main/interface/cli.c

```diff
@@ -2443,7 +2443,7 @@ static void cliPrintGyroRegisters(uint8_t whichSensor)
 
 static void cliDumpGyroRegisters(char *cmdline)
 {
-#ifdef USE_DUAL_GYRO
+#ifdef GYRO_2_CS_PIN
```

Comment:
Not sure if it's a good idea to replace the `USE_DUAL_GYRO` with a pin define that we know will have to go again before we are ready for universal targets.